lang/cs/Org.Apache.REEF.Client/API/IREEFClient.cs

```diff
@@ -50,5 +51,12 @@ public interface IREEFClient
         /// <returns></returns>
         [Unstable("0.14", "Working in progress for rest API status returned")]
         Task<FinalState> GetJobFinalStatus(string appId);
+
+        /// <summary>
+        /// Returns all the application reports running in the cluster
+        /// </summary>
+        /// <returns></returns>
+        [Unstable("0.16", "Working in progress for rest API id returned")]
+        Task<List<IApplicationReport>> GetApplicationReports();
```

Comment:
This adds a new API to the generic REEF client. Would it make sense to move this to a specific YARN client instead?